.github/workflows/main-build.yml

```diff
@@ -46,6 +46,16 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
+#      # adot customizations, such as replace collector to aoc, xray backend collector config, etc.
+#      - name: Patch ADOT
+#        run: |
+#          cp -rf adot/* opentelemetry-lambda/
```

Comment:
Let's extract `patch-upstream.sh` for this

Reply:
will add a patch shell with unblocking adot customization test.